### ISSUE TITLE
Security middleware initialization fails with operation without medthodDoc.

### ISSUE THREAD

**Issue by MugeSo** [kogosoftwarellc/express-openapi#35]
Other middleware initialization in if block begins in L.130.
```javascript
      if (methodDoc &&
          allowsMiddleware(apiDoc, pathModule, pathItem, methodDoc)) {// add middleware
```

Security middleware is initialized out of the block.

**Comment by jsdevel** [comment]
dang.  You're right!

**Comment by jsdevel** [comment]
Fixed in `v0.20.2`
